contracts/SignerBondsUniswapV2.sol

```diff
@@ -77,6 +77,9 @@ contract SignerBondsUniswapV2 is ISignerBondsSwapStrategy, Ownable {
     address public immutable assetPool;
     address public immutable collateralToken;
 
+    // Currently approved operators that can swap signer bonds on Uniswap V2
```

Comment:
Can we drop "Currently"? We do not have a list of pending approvals, so I think *currently* is misleading.

Reply:
I removed the comment entirely.